src/main/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueue.java

```diff
@@ -70,6 +67,7 @@
 public class SingleChronicleQueue extends AbstractCloseable implements RollingChronicleQueue {
 
     public static final String SUFFIX = ".cq4";
+    public static final String DISCARD_FILE_SUFFIX = ".discard";
```

Comment:
I would suggest a sequence or timestamp-based decoration is a better/more flexible option

Reply:
Timestamp-based decoraton means we may copy a single cycle an arbitrary number of times, which in case of high performance system might as well be tens of thousands times which will exhaust inodes and-or current directory files limit. That's why I advise we use predictable discard name and avoid this issue entirely. If it's not successful after first try it's probably too dangerous to continue - what do you think @rogersimmons ?

Reply:
Yes, I agree with the concern, and it is something we should be in a position to mitigate.
Having a sequence/time-based decoration helps rather than impedes that, and at the same time gives us more control options in future if needed

Reply:
I don't know how we could mitigate this *right now*, whereas the current solution is indeed limited but is predictable and completely understood, and we can reason about its behavior.

I suggest that maybe we add time-based decoration together with the change that protects about creating the same cycle file unbounded number of times.